### PR TITLE
Documentation cleanup

### PR DESCRIPTION
**Pull request by Doczillar**
## Types of changes

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x ] I have read the **[CONTRIBUTING](/CONTRIBUTING.md)** document
- [ x] My code follows the code style of this project (hint: install an [xo editor plugin](https://github.com/xojs/xo#editor-plugins))
- [ x] Any required documentation has been added
- [ x] I have added tests to cover my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,20 +12,49 @@
 <a href="https://smartthingsdev.slack.com/messages/CG595N08N"><img src="https://badgen.net/badge//smartthingsdev?icon=slack"/></a>
 </p>
 
-#### [Reference Documentation](docs/index.md)
+## Reference Documentation
 
-SDK that wraps the SmartThings REST API and reduces the amount of code necessary to write a SmartApp app. It supports 
-both webhook and AWS Lambda implementations. This is a preview version of the API and will change over time time.
+Looking for reference documentation? Find it [here](docs/index.md).
```
[the remark this answers is not in the document]
Comment (john-u) [reply README.md:17]: How about we phrase it something like this:

> For a detailed look at the API, see the [reference documentation]().

File: README.md [hunk 2]
```diff
@@ -12,20 +12,49 @@
...
-_ATTENTION! This major release is not completely backwardly compatible with the 1.X version, though for most
-SmartApps the changes required should be relatively minor. The major non-backwardly changes are:_
-* _Methods that return lists now return arrays rather that an object with the properties `items` and `_links`._
-* _Axios is now used rather than request-promise-native for making HTTP calls, resulting in changes to the error
+>**Version 2.0 Release**
+>
+>This is a preview version of the API and will change over time.
+>
+>*This major release is not completely backwardly compatible with  version 1.X*, though for most SmartApps the changes required should be relatively minor. The major non-backwards compatible changes include:
+>* _Methods that return lists now return arrays rather that an object with the properties `items` and `_links`._
```
Comment (john-u) [inline README.md:26]: ~rather that an object~ instead of objects

File: README.md [hunk 3]
```diff
@@ -12,20 +12,49 @@
...
+>* _Axios is now used rather than request-promise-native for making HTTP calls, resulting in changes to the error
 objects thrown when exceptions occur._
+>
+>See the [Version 2.0.0 release notes](docs/V2_RELEASE_NOTES.md) for more information.
 
-_See the [Version 2.0.0 release notes](docs/V2_RELEASE_NOTES.md) for more information._
+This SDK includes a set of NodeJS libraries for building Webhook and AWS Lambda SmartApps, and interacting with the public SmartThings API.
```
Comment (john-u) [inline README.md:32]: A nit, but `Node.js` is a more popular way of writing it.

Comment (Doczillar) [reply README.md]: Revised, thanks for catching that.

**Review by john-u — APPROVED** [review]

**Review by rossiam — APPROVED** [review]

**Pull request merged**